### DISCUSSION TITLE
Coaching

### DISCUSSION THREAD

**Discussion by Cooltilcr** [community/community#133016] [Programming Help] [answered]
### Body

Someone who can help to understand what's going on with tutorials and security issues 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Cooltilcr** [comment]
There's a lot of YouTubers are going on it

**Comment by ebndev** [comment, marked as answer]
Hi @Cooltilcr, thanks for being a part of the GitHub Community!

Please note: we currently do not allow job listings or solicitation on Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

In the meantime, if you have any specific questions about what you're learning, please feel free to post again in our `Programming Help` category!
